pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -116,13 +116,7 @@ def plugin2nix(self, pdesc: PluginDesc, plugin: pluginupdate.Plugin) -> str:
 def main():
 
     global luaPlugins
-
-    # whitelist
-    luaPlugins = run_nix_expr(GET_PLUGINS_LUA) + [
-        "diffview-nvim",
-        "marks-nvim",
-        "nvim-biscuits"
-    ]
+    luaPlugins = run_nix_expr(GET_PLUGINS_LUA)
```

Comment:
I dont understand how it is still there, most likely I forgot to commit those removal back (they dont exist in my fork). Well spotted.